Makefile

```diff
@@ -0,0 +1,6 @@
+build-sample:
```

Comment:
Any reason we have this make file, could this not just be a shell script. Reduce the technologies used.